### ISSUE TITLE
How to define a custom loss function of  multi-class logarithmic loss?

### ISSUE THREAD

**Issue by 157995010** [fchollet/keras#4406]
I need to define a custom loss function of multi-class logarithmic loss, the formula is show in the picture.
![image](https://cloud.githubusercontent.com/assets/9782784/20377212/84f15630-acc8-11e6-88b0-4dbff7ed4539.png)
Just like the ```def categorical_crossentropy(y_true, y_pred):```, I wonder is  `y_true` an array of softmax probability, and `y_pred ` an array of one-hot encoded array. Can I just use numpy to calculate the loss value without use the `backend (K)`.
I am a newbie, I hope someone can tell me how to write a function of that loss function in the figure?

**Comment by avatar-lavventura** [comment]
@157995010 Did you able to find solution to your question?
